data/mods/Aftershock/items/tools.json

```
@@ -315,5 +315,24 @@
     "revert_to": "control_laptop",
     "use_action": { "target": "control_laptop", "msg": "You stop lighting up the screen.", "menu_text": "Turn off", "type": "transform" },
     "flags": [ "WATCH", "LIGHT_10", "TRADER_AVOID" ]
+  },
+  {
+    "id": "alien_battery",
+    "type": "MAGAZINE",
+    "category": "spare_parts",
+    "name": { "str": "alien cell", "str_pl": "alien cells" },
```

```suggestion
    "name": { "str": "alien cell" },
```
"Cells" can be automatically generated, according to the JSON validation on Travis.